alembic/versions/ebf7613506e0_initial_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Initial migration
+
+Revision ID: ebf7613506e0
+Revises: 
+Create Date: 2024-08-18 23:29:03.639693
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'ebf7613506e0'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
```

Comment:
does alembic need support for python <3.10? if this is autogenerated, can ignore

Reply:
It's auto generated